sentry-spring-boot-starter/build.gradle.kts

```diff
@@ -0,0 +1,117 @@
+import com.novoda.gradle.release.PublishExtension
+
+plugins {
+    `java-library`
+    kotlin("jvm")
+    jacoco
+    id(Config.QualityPlugins.errorProne)
+    id(Config.Deploy.novodaBintray)
+    id(Config.QualityPlugins.gradleVersions)
+    id(Config.BuildPlugins.buildConfig) version Config.BuildPlugins.buildConfigVersion
+    id(Config.BuildPlugins.springBoot) version Config.springBootVersion apply false
+}
+
+apply(plugin = Config.BuildPlugins.springDependencyManagement)
+
+the<io.spring.gradle.dependencymanagement.dsl.DependencyManagementExtension>().apply {
+    imports {
+        mavenBom(org.springframework.boot.gradle.plugin.SpringBootPlugin.BOM_COORDINATES)
+    }
+}
+
+configure<JavaPluginConvention> {
+    sourceCompatibility = JavaVersion.VERSION_1_8
+    targetCompatibility = JavaVersion.VERSION_1_8
+}
+
+tasks.withType<org.jetbrains.kotlin.gradle.tasks.KotlinCompile>().configureEach {
+    kotlinOptions.jvmTarget = JavaVersion.VERSION_1_8.toString()
+}
+
+dependencies {
+    api(project(":sentry-core"))
+    implementation(Config.Libs.springBootStarter)
+    implementation(Config.Libs.springWeb)
+    implementation(Config.Libs.servletApi)
+
+    annotationProcessor("org.springframework.boot:spring-boot-autoconfigure-processor")
+    annotationProcessor("org.springframework.boot:spring-boot-configuration-processor")
```

Comment:
can we also add to the `Config` class?

Reply:
have you tried to use kapt? https://kotlinlang.org/docs/reference/kapt.html

Reply:
I haven't but I also wouldn't like to opt-out from Spring Boot recommended way of creating starters.

Reply:
I see, well if it works, its fine, I know that on Android/Gradle if you have the Kotlin plugin applied, `kapt` is always preferred, because `annotationProcessor` won't work for Kotlin code anyway, as we don't have annotation processing on Android right now nor we use Kotlin code (only on tests), it's fine.